src/seedsigner/views/seed_views.py

```diff
@@ -583,7 +590,10 @@ def run(self):
             button_data.append(self.EXPORT_XPUB)
 
         button_data.append(self.EXPLORER)
-        button_data.append(self.BACKUP)
+
+        if not isinstance(self.seed, ShamirSeed):
+            #TODO: enable to backup just one share. Extendable flag must be supported by embit
+            button_data.append(self.BACKUP)
```

Comment:
In general this is fine.

But I dislike custom logic just for one type of `Seed` when we do already have a mechanism in there to turn features on and off (e.g. `self.seed.bip85_supported`). Ideally a dev should be able to be completely blissfully unaware of any Shamir details if they haven't had a need to dig into it (well, okay, being totally unaware of a whole chunk of code can be dangerous, but we also have the test suite to catch any issues if a naive dev inadvertently broke something in Shamir without knowing it).

And so I dislike it even more that there's an even deeper in-the-weeds Shamir TODO comment (which I still don't fully understand).

Given all this, I think I'd prefer a `self.seed.backup_supported` approach.